packages/shared/forks/ReactFeatureFlags.www-dynamic.js

```diff
@@ -25,6 +25,7 @@ export const enableUnifiedSyncLane = __VARIANT__;
 export const enableTransitionTracing = __VARIANT__;
 export const enableCustomElementPropertySupport = __VARIANT__;
 export const enableDeferRootSchedulingToMicrotask = __VARIANT__;
+export const diffInCommitPhase = __VARIANT__;
```

Comment:
@kassens I prepped this for www since it would be good to test that this doesn't regress asap.